mobile/.expo/settings.json

```diff
@@ -3,5 +3,5 @@
   "lanType": "ip",
   "dev": true,
   "minify": false,
-  "urlRandomness": "66-qcw"
+  "urlRandomness": "2s-bjc"
```

Comment:
Should this have notes around why this needs to be this specific thing?

Reply:
It's automatically generated by Expo